languages/python/templates/requirements.mustache

```diff
@@ -1,5 +1,8 @@
 {{! This file is picked from the v5.1.0 release of openapi-generator(https://github.com/OpenAPITools/openapi-generator/blob/v5.1.0/modules/openapi-generator/src/main/resources/python/requirements.mustache) }}
-
+{{! Changes made: 1) Added requirements for fds.protobuf.stach and pandas. }}
 python_dateutil >= 2.5.3
 setuptools >= 21.0.0
 urllib3 >= 1.25.3
+fds.protobuf.stach.extensions < 2.0.0
+pandas < 2.0.0
+openpyxl >= 3.0.0
```

Comment:
If these two are `stach-extensions`'s dependencies, can you please remove them here?

Reply:
When they didn't exist, examples that used stach extensions wouldn't run properly

Reply:
I think that you can remove pandas. That should come as a part of `stach extensions` package. Basically, these can create possible issues in the future for us. 
`openpyxl` should be a dependency for `examples` project directly.

Reply:
Can you please remove `pandas` in the `setup.py` too?